dataland-frontend/src/components/forms/CreateSFDRDataset.vue

```diff
@@ -0,0 +1,330 @@
+<template>
```

Comment:
Filename violates camelcase convention.   In other names for Sfdr stuff we always write "...Sfdr..."

Reply:
Agree, added a to do to keep track of this, as i would prefer to rename this file once nobody else is working on it.

Reply:
Ok then let's do it at the end.  Will keep this conversation open as reminder.

Reply:
But why waiting? This is also a new file so renaming it should be totally ok for git.

Reply:
Will rename it in the next commit

Reply:
👍